shared/actions/platform-specific/index.native.tsx

```diff
@@ -27,7 +27,9 @@ import {Permissions} from 'react-native-unimodules'
 import {isIOS, isAndroid} from '../../constants/platform'
 import pushSaga, {getStartupDetailsFromInitialPush} from './push.native'
 import ImagePicker from 'react-native-image-picker'
-import {TypedActions, TypedState} from 'util/container'
+import {TypedActions, TypedState} from 'util/container' // how does this work?
```

Comment:
The import is from `util/container` and not `../../util/container`

Reply:
TS is fine with using these types, but if I switch this to import actual code then this doesn't work anymore

Reply:
oh, hmm, ts magic

Reply:
are the types actually valid

Reply:
Yeah the types are good. I think TS finds it because of `compilerOptions.baseUrl` in tsconfig.base, judging by [this](https://stackoverflow.com/questions/42123382/import-module-from-root-path-in-typescript)

Reply:
Does it get compiled correctly into relative imports?

Reply:
No @jzila see above

> if I switch this to import actual code then this doesn't work anymore

Reply:
Fair enough. I guess we can't use them for now?